modules/product-catalog/src/generateSchema.ts

```diff
@@ -47,10 +55,12 @@ const generateZuoraProductSchema = (product: CatalogProduct) => {
 	const ratePlanSchema = supportedRatePlans.map((productRatePlan) =>
 		generateProductRatePlanSchema(productRatePlan),
 	);
+	const isDeliveryProduct = deliveryProducts.includes(productName);
```

Comment:
I got a type error here, perhaps this would work?
```suggestion
	const isDeliveryProduct = deliveryProducts.includes(productName);
```